recipes/python-glmnet/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "glmnet" %}
+{% set version = "2.0.0" %}
+{% set md5 = "e0465e76af8c22440d086f34d9ad8d5c" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - pip
+    - python
+    - setuptools
+    - numpy x.x
+    - scikit-learn >=0.18.0
+    - scipy >=0.14.1
+    - gcc  # [(linux or osx) and not win]
+
+  run:
+    - numpy x.x
+    - scikit-learn >=0.18.0
+    - scipy >=0.14.1
+    - libgcc  # [linux]
+    - libgfortran  # [osx]
+
+test:
+  imports:
+    - glmnet
+
+about:
+  home: https://github.com/civisanalytics/python-glmnet
+  license: GPL-2.0
+  summary: 'Python wrapper for glmnet'
+  dev_url: https://github.com/civisanalytics/python-glmnet
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    jseabold
```

Comment:
FYI this was missing a `-` before it. Have raised an issue in the feedstock regarding getting this fixed.

xref: https://github.com/conda-forge/glmnet-feedstock/issues/1